index.js

```diff
@@ -3,38 +3,173 @@ const app = express();
 const config = require("./src/lib/config");
 const db = require("./src/lib/db");
 
-const User = require("./src/models/user").model;
-const Pickup= require("./src/models/pickup").model;
+const userUsecases = require("./src/usecases/user");
+const packageUsecases = require("./src/usecases/package");
+const pickupUsecases = require("./src/usecases/pickup");
+const zoneUsecases = require("./src/usecases/zone");
 
 app.use(express.json());
 
 // Ejecutando el servidor HTTP
 app.listen(config.app.port, async () => {
   console.log(`Esuchando peticiones HTTP en el puerto ${config.app.port}`);
-
+  
   try {
     await db.connect();
     console.log("DB is connected 🤠");
+
+    // const newUser = await userUsecases.create({email: "nuevo@asdf.com", password: "test1", firstName: "alguien", lastName: "más", phone: "1234123412", street: "irolo", number: 123, interior: 401, neighborhood: "zaca", municipality: "benito juarez", state: "cdmx", zipCode: 4000});
+    // console.log(newUser);
+    const userById = await userUsecases.findByEmail("nuevo@asdf.com");
```

Comment:
Hardcoded value, this should be sent in the body of the request. Plus this file is full of commented lines.